src/payment/strategies/affirm/affirm-payment-strategy.ts

```diff
@@ -0,0 +1,283 @@
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { MissingDataError, MissingDataErrorType, NotInitializedError, NotInitializedErrorType } from '../../../common/error/errors';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { PaymentArgumentInvalidError, PaymentMethodCancelledError } from '../../errors';
+import PaymentActionCreator from '../../payment-action-creator';
+import PaymentMethodActionCreator from '../../payment-method-action-creator';
+import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
+import PaymentStrategy from '../payment-strategy';
+
+import { Affirm, AffirmAddress, AffirmDiscount, AffirmItem, AffirmRequestData, SuccessAffirm } from './affirm';
+import AffirmScriptLoader from './affirm-script-loader';
+
+export default class AffirmPaymentStrategy implements PaymentStrategy {
+
+    private affirm?: Affirm;
```

Comment:
Could you put `_` prefix for private properties? The reason is because, although we're using TS, we still want to indicate which properties are intended to be private when the code is compiled to JS.